site/public/css/server.css

```diff
@@ -364,6 +364,11 @@ table tr.table-header {
     }
 }
 
+.gradeable_message {
+    background-color: #006398 !important; 
```

Comment:
Instead of using `!important` just make the selector more specific eg `.content.gradeable_message`